02-installation.md

```diff
@@ -108,6 +108,10 @@ The backend requires 3 exposed ports and persistent storage. This example uses a
 
 We suggest, but do not require, persistent storage for sensu-agents. The Sensu Agent will cache runtime assets locally for each check (see [Checks and Assets](06-checks-and-assets.md) for more details). This storage should be unique per sensu-agent process.
 
+### Dashboards and Visualizations
+
+Currently Uchiwa is not yet supported but it is under active development, with a release currently planned for late 2018.
```

Comment:
Let's not add a release date. An initial dashboard is likely to come out sooner than that. Also, the dashboard will be built directly into the sensu-backend process instead of a separate service. I do not know if @palourde has plans to port Uchiwa to Sensu 2.0. Simon?

Reply:
I don't personally have any plans to add support for Sensu 2.0 into Uchiwa, but I would definitively accept and assist anyone interested in doing so!